File: MouseInvasion/app/src/main/java/br/odb/mouseinvasion/CheeseDefenderApplication.java
package br.odb.mouseinvasion;

import android.app.Application;
import android.content.Context;
import android.media.AudioManager;

public class CheeseDefenderApplication extends Application  {
	boolean mayEnableSound() {
		AudioManager am = (AudioManager)this.getSystemService(Context.AUDIO_SERVICE);

		if (am.getRingerMode() == AudioManager.RINGER_MODE_NORMAL) {
			return true;
		} else {
			return false;
		}
	}

}

File: MouseInvasion/app/src/main/java/br/odb/mouseinvasion/CheeseDefenderApplication.kt
package br.odb.mouseinvasion

import android.app.Application
import android.media.AudioManager

class CheeseDefenderApplication : Application() {
	fun mayEnableSound(): Boolean {
		val am = this.getSystemService(AUDIO_SERVICE) as AudioManager
		return (am.ringerMode == AudioManager.RINGER_MODE_NORMAL)
	}
}